projects/pumpspace/index.js

```diff
@@ -0,0 +1,113 @@
+// projects/pumpspace/index.js
+const ADDRESSES = require('../helper/coreAssets.json')
+const { staking } = require('../helper/staking')
+const { sumTokens2 } = require('../helper/unwrapLPs')
+const sdk = require('@defillama/sdk')
+
+const CHAIN = 'avax'
+
+// --- FACTORIES / CONTRACTS ---
+const PUMP_FACTORY   = '0x26B42c208D8a9d8737A2E5c9C57F4481484d4616' // V2
+const PUMP_V3        = '0xE749c1cA2EA4f930d1283ad780AdE28625037CeD' // V3/Trident
+
+// If you later expose staking for other MasterChefs, add here
+const MASTERCHEFS = [
+  '0x40a58fc672F7878F068bD8ED234a47458Ec33879', // SHELL
+  '0x56b54a1384d35C63cD95b39eDe9339fEf7df3E42', // KRILL
+  '0x06C551B19239fE6a425b3c45Eb8b49d28e8283C6', // PEARL
+]
+
+// --- TOKENS (project/local wrappers + protocol tokens) ---
+const TOKENS = {
+  SHELL: '0xaD4CB79293322c07973ee83Aed5DF66A53214dc6',
+}
+
+// ---------- V2 factory TVL (sum token reserves held by each pair) ----------
+const uniAbis = {
+  allPairsLength: 'function allPairsLength() view returns (uint256)',
+  allPairs:       'function allPairs(uint256) view returns (address)',
+  token0:         'function token0() view returns (address)',
+  token1:         'function token1() view returns (address)',
+}
+
+// ---------- V3 (Trident) TVL: factory → pool list → asset list → sum with transforms ----------
+const tridentAbis = {
+  totalPoolsCount: "uint256:totalPoolsCount",
+  getPoolAddress: "function getPoolAddress(uint256 idx) view returns (address pool)",
+  getAssets: "address[]:getAssets",
+}
+
+
+async function v2FactoryTVL(api) {
```

Comment:
please check `const { getUniTVL } = require('./helper/unknownTokens')` I think there is code duplication here